library/nuxeo

```diff
@@ -1,8 +1,9 @@
 # maintainer: Damien Metzler <dmetzler@nuxeo.com> @dmetzler
 
-latest: git://github.com/nuxeo/docker-nuxeo@09c50e490336f28caf1278c6bffaa02e79c0f4b4 8.1
-FT: git://github.com/nuxeo/docker-nuxeo@09c50e490336f28caf1278c6bffaa02e79c0f4b4 8.1
-8: git://github.com/nuxeo/docker-nuxeo@09c50e490336f28caf1278c6bffaa02e79c0f4b4 8.1
+latest: git://github.com/nuxeo/docker-nuxeo@c8f5d5bd4fedda1ee72eb29eac5fbb0b0a490155 8.2
+FT: git://github.com/nuxeo/docker-nuxeo@c8f5d5bd4fedda1ee72eb29eac5fbb0b0a490155 8.2
+8: git://github.com/nuxeo/docker-nuxeo@c8f5d5bd4fedda1ee72eb29eac5fbb0b0a490155 8.2
+8.2: git://github.com/nuxeo/docker-nuxeo@c8f5d5bd4fedda1ee72eb29eac5fbb0b0a490155 8.2
 8.1: git://github.com/nuxeo/docker-nuxeo@09c50e490336f28caf1278c6bffaa02e79c0f4b4 8.1
```

Comment:
Not any particular reason. 8.1 and 7.4 should not evolve as these are fastrack versions. I will remove them from the build file